decidim-meetings/lib/decidim/meetings/component.rb

```diff
@@ -57,6 +57,9 @@
     settings.attribute :resources_permissions_enabled, type: :boolean, default: true
     settings.attribute :enable_pads_creation, type: :boolean, default: false
     settings.attribute :creation_enabled_for_participants, type: :boolean, default: false
+    settings.attribute :citizen_map_pin_color, type: :string
```

Comment:
These setting should be at the organization level rather than the component, otherwise admins will have to configure it each time.

Reply:
Added as dependencies to OrganizationAppearanceForm, and they are set as general highlight color that can be used in many places. At the moment the map pin personalization, can be used as a starting point. 

      attribute :group_highlight_enabled, Boolean
      attribute :official_highlight_color, String, default: "#ef604d"
      attribute :group_highlight_color, String, default: "#599aa6"
      attribute :citizen_highlight_color, String, default: "#57d685"